Jessy/email_ids_challenge.py

```diff
@@ -0,0 +1,52 @@
+# You run a startup media company called Ripple Media
+# It's typical when you hire a new employee in your company, to setup an email id for them
+
+print('Question 1')
+employee_name = 'Ash Rahman'
+
+# You have decided the format of the email should be: Ash Rahman -> ash.rahman@ripplemedia.com
+# Let's write some code that converts a name into an email id that matches this format
+# 1.1 TODO: Let's save the lowercase version of the employee_name in a new variable 'lower_name'
+lower_name= employee_name.lower()
+print(lower_name)
+# I simply used the conversion function to lower all the letters.
+# 1.2 TODO: We want to separate the first name and last name and save it in a variable 'names_list'
+name_list = lower_name.split()
+print(name_list)
+
+# 1.3 TODO: We want to join the first name and last name with a '.' and save it in a variable called 'joined_names'
+# joined_names
+# This was tough, had to look for videos online. I don't remember viewing .split, .join videos on Canvas.....But I love it1!!
+joined_names='.'.join(name_list)
+print(joined_names)
+# 1.4 TODO: We want to add '@ripplemedia.com' to the end of the string inside joined_names and save it in a variable 'email'
+# I had a hard time with this question
+email=joined_names+"@ripplemedia.com"
+print(email)
+print()
+
+print('Question 2')
+
+# Congratulations! Your team is expanding. Below is a list of their names:
+names = ['Max Bartlett', 'Angelita Norris', 'Stewart Mueller', 'Dominique Henry', 'Carmela Gross', 'Bettie Mcmillan', 'Sara Ellison', 'Ira Anthony', 'Pauline Riley', 'Ben Weber',
+         'Joanne Mcknight', 'Loren Gould', 'Jamar Singh', 'Amanda Vance', 'Tyrell Andrade', 'Jana Clements', 'Eddy Mcbride', 'Marsha Meyer', 'Elbert Shannon', 'Alyce Hull']
+
+emails = []
+for i in names:
+    emails.append(i+"@ripplemedia.com")
+
+print(emails)
+# We want to convert all their names into the same format from Question 1
+# 2.1 TODO: Use a "for" loop to go over each name in the names list
+for expansion in names:
+    print(expansion)
+# print()
+# 2.2 TODO: Inside the "for" loop, create the email id by re-using the logic from Question 1 and...
+
+for ripplemedia in names:
```

Comment:
Here is the breakdown from the answer key

emails = [ ]

for name in names:
1### lower_name = name.lower()        
2### names_list = lower_name.split(' ')        
3### joined_names = '.'.join(names_list)       
4###email = joined_names + '@ripplemedia.com'     
5###emails.append(email)      


 1) takes each name and lowers
2) splits name, the parameter added here was a space value,
3) joins the split with a period " . "
4) adds @ripplemedia.com string
5) appends to list